scripts/minimal_init.lua

```diff
@@ -13,6 +13,7 @@ vim.filetype.add {
     usda = "usd",
     wgsl = "wgsl",
     w = "wing",
+    promql = "promql",
```

Comment:
```suggestion
```
Not needed.

Reply:
without it, `.promql` files are not recognized as `promql` filetype and newly added unit tests involving `.promql` files fail. On the other hand, unit tests with the `.yaml` files containing `promql` queries finish successfully because `yaml` injections are correctly recognized being `promql`. Is there a way to workaround the issue?

Reply:
Fixed using `vim: ft=promql` in test files

Reply:
Yes, that's cleaner.

Reply:
Ideally, someone would contribute filetype detection for promql files to Vim. (No need for syntax or other files; just detection is enough.)

Reply:
Interestingly, I never encountered standalone `*.promql` files in the wild, they always live as a part of some yaml configuration or template. There are few examples on GitHub (https://github.com/search?q=path%3A*.promql%20NOT%20is%3Aarchived%20NOT%20is%3Afork&type=code), but probably it's not the best metric because it's not very common to see open-sourced infrastructure code on GitHub anyway

Reply:
Makes sense; then the modeline is indeed the best approach.

Reply:
So this was initially written for Alertmanager config files; i think i might have added the FT just for the tests maybe.